dfdaemon/config/config.go

```diff
@@ -20,19 +20,20 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"encoding/json"
-	"io/ioutil"
 	"net/url"
 	"os"
 	"path/filepath"
 	"regexp"
 
 	dferr "github.com/dragonflyoss/Dragonfly/common/errors"
-	"github.com/dragonflyoss/Dragonfly/common/util"
 	"github.com/dragonflyoss/Dragonfly/dfdaemon/constant"
+	"github.com/spf13/afero"
```

Comment:
Please construct the structure of your importing packages according to the rule:

// Golang internal package

// Dragonfly internal package

// Third-party package

Reply:
fixed